### ISSUE TITLE
manifest embedded, linked, both?

### ISSUE THREAD

**Issue by TzviyaSiegman** [w3c/wpub#25]
Should the manifest be in an external file, embedded in a specified manner, or should either option be allowed?

**Comment by lrosenthol** [comment]
I think we need to further understand what will be in the manifest and what it might look like before we can answer this...but definitely worth tracking this...

**Comment by mattgarrish** [comment]
We're not the first ones to discuss this, either:

https://gist.github.com/marcoscaceres/7783977

It's interesting that by virtue of using ```link``` it does allow inlining through a data: url if you really want to.

Would that be a useful approach for FPWD, with this being one of the questions we should highlight for feedback?

**Comment by rdeltour** [comment]
> We're not the first ones to discuss this

correct, see also [Appendix A of the Web App Manifest spec](https://www.w3.org/TR/appmanifest/#relationship-to-html%27s-link-and-meta-elements) which also points to a [github comment](https://github.com/w3c/manifest/issues/97) and [www-tag thread](https://lists.w3.org/Archives/Public/www-tag/2014Jan/0139.html) discussing the matter.

**Comment by iherman** [comment]
@mattgarrish:

> It's interesting that by virtue of using link it does allow inlining through a data: url if you really want to.

Yeah, although… that would probably be very error prone. Note the comment of Tobie on the possibility to use a meta element; I am afraid (though have not checked) that most of what he says there is also valid for the data url case:

https://gist.github.com/marcoscaceres/7783977#gistcomment-963994

I do not think we should propagate that.

Clearly, the reactions were extremely varied. Note, however, that we may be dependent on the outcome of issue #32: if we go with an extension to the Web App Manifest, then we will not have the liberty of making a choice.

**Comment by mattgarrish** [comment]
> I do not think we should propagate that.

Right, I'm not a big fan of embedding. I don't think we should say anything about it formally, just as web app manifest doesn't. But it is there for the intrepid.

**Comment by iherman** [comment]
Closing this in favor of #122
